addons/website_sale_coupon/views/website_sale_templates.xml

```diff
@@ -6,6 +6,43 @@
         <attribute name="t-if">not line.is_reward_line</attribute>
     </xpath>
 </template>
+<template id="layout" inherit_id="website.layout">
+    <body position="inside">
+        <t t-set="promo_code" t-value="request.session.get('pending_coupon', {}).get('promo_code')"/>
+        <t t-if="request.params.get('notify_coupon')">
+            <t t-set="error" t-value="request.session.get('pending_coupon', {}).get('error') or request.session.get('pending_coupon', {}).get('not_found')"/>
```

Comment:
Dont name a `t-set` variable `error`, this is a nightmare to grep and the best way to have collision
`pending_coupon_error` would be better